src/browser/main.ts

```diff
@@ -0,0 +1,68 @@
+import { desktopCapturer, remote } from "electron";
+import pino from "pino";
+import {
+  K_BROWSER_CONFIG,
+  K_BROWSER_STORAGE,
+  K_CAPTURE_WIN,
+  K_PRELOAD_INIT_KEY,
+  K_SIGNAL_CONFIG,
+} from "../base/constants";
+import { Application } from "./application";
+import { H264Sdp } from "./h264-sdp";
+import { Input } from "./input";
+import { Peer } from "./peer";
+import { Signal } from "./signal";
+import { UserMedia } from "./usermedia";
+
+// we'll export this and use it for testing
+// it won't impact the runtime as the runtime ignores it
+let runtimeIgnoredExportValue: Promise<void>;
+
+const logger = pino();
+const config = remote.getGlobal(K_BROWSER_STORAGE);
+
+if (!config[K_PRELOAD_INIT_KEY]) {
```

Comment:
Do we add this new logic for debugging purposes? It's not clear to me how the preload script of the streamer window can be invoked more than once.

Reply:
ctrl+r in the streamer window :)

it's really an extra precaution, not a real-world scenario.